Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -166,9 +173,10 @@ - (dispatch_queue_t)methodQueue
     RCTLogError(@"Not a value node.");
     return;
   }
+
   RCTValueAnimatedNode *valueNode = (RCTValueAnimatedNode *)node;
   valueNode.value = value.floatValue;
-  [valueNode setNeedsUpdate];
+  [_updatedValueNodes addObject:valueNode];
```

Comment:
Why this change? Seems like we never added to `_updatedValueNodes` before? Why remove the call to `setNeedsUpdate`?